cmake/linker/ld/target.cmake

```
@@ -167,7 +167,7 @@ macro(toolchain_linker_finalize)
     if(CONFIG_CPP_EXCEPTIONS AND LIBGCC_DIR)
       # When building with C++ Exceptions, it is important that crtbegin and crtend
       # are linked at specific locations.
-      set(cpp_link "<LINK_FLAGS> ${LIBGCC_DIR}/crtbegin.o ${link_libraries} ${LIBGCC_DIR}/crtend.o")
+      set(cpp_link "${LIBGCC_DIR}/crtbegin.o ${link_libraries} ${LIBGCC_DIR}/crtend.o")
```

please restore, as per other comments:
```suggestion
      set(cpp_link "<LINK_FLAGS> ${LIBGCC_DIR}/crtbegin.o ${link_libraries} ${LIBGCC_DIR}/crtend.o")
```